src/style.css

```diff
@@ -0,0 +1,29 @@
+body {
+  font: 13px Arial;
+}
+
+h1,
+h2,
+p {
+  margin: 0;
+}
+
+ol {
+  margin-top: 0;
+  margin-bottom: 0;
+}
```

Comment:
you have the same styles for margin above, thinks about how to combine them together